pkg/alertmanager/alertmanager_metrics.go

```diff
@@ -168,6 +175,34 @@ func newAlertmanagerMetrics() *alertmanagerMetrics {
 			"cortex_alertmanager_state_replication_failed_total",
 			"Number of times we have failed to replicate a state to other alertmanagers",
 			[]string{"user"}, nil),
+		fetchReplicaStateTotal: prometheus.NewDesc(
+			"cortex_alertmanager_fetch_replica_state_total",
```

Comment:
```suggestion
			"cortex_alertmanager_state_fetch_replica_state_total",
```

Reply:
Even if it stutters I think it's best if we keep the prefix consistent, when looking for what metrics to explore this is quite helpful.

Reply:
Agreed, fixed.